src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -70,12 +71,17 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
         $this->itemMutationResolverFactory = $itemMutationResolverFactory;
         $this->defaultFieldResolver = $defaultFieldResolver;
         $this->filterLocator = $filterLocator;
+        $this->typesFactory = $typesFactory;
         $this->paginationEnabled = $paginationEnabled;
     }
 
     public function getSchema(): Schema
     {
-        $this->graphqlTypes['Iterable'] = new IterableType();
+        if ($this->typesFactory) {
+            $this->graphqlTypes += $this->typesFactory->getTypes();
+        } else {
```

Comment:
Should we remove this `else` or add a deprecation?

Reply:
IIRC this is still marked as Experimental no? I'd just keep the l81.

Reply:
I was asking this because of @dunglas's comment: https://github.com/api-platform/core/pull/2447#discussion_r248777048